CHANGES.md

```diff
@@ -1,6 +1,12 @@
 # Changes
 
 This document describes the relevant changes between releases of the API model.
+
+## 0.0.319 September 19 2023
```

Comment:
Unfortunately, the PR https://github.com/openshift-online/ocm-api-model/pull/838 is grabbing 0.0.319 release. I'll let you know if that's one merged before yours so that you can update the version here or if you prefer you can do it separately.

Reply:
This is documented here https://github.com/openshift-online/ocm-sdk-go/blob/main/CONTRIBUTING.md#releasing-a-new-ocm-api-model-version

Reply:
The PR https://github.com/openshift-online/ocm-api-model/pull/838 has been merged. You'll need to bump the version again. Or alternatively you could separate introducing the changes and bumping the version onto two PRs.

Reply:
Thanks for the pointer @machi1990 . I'll remove the version details from my PR. Makes complete sense to do it when we cut a release after the PR is merged.